microsoftgraph/microsoft-graph-explorer-v2
# Feat/query runner

## Overview

1. Adds the QueryRunner component

### Demo

![queryrunner component](https://user-images.githubusercontent.com/12011447/53319078-86fcf580-38e2-11e9-9b0c-08c926f4fe30.JPG)

![response](https://user-images.githubusercontent.com/12011447/53319084-89f7e600-38e2-11e9-84ed-a47e1393056b.JPG)

### Notes

This PR does not implement 
1. Accessibility
2. Localization
3. Support for POST, PATCH, DELETE & PUT
4. Authentication

The features named above are not necessary for the prototype. They will be included when we start working towards the MVP.

## Testing Instructions

* Pull the changes
* Run `npm install`
* Run `npm test` to makes sure all tests pass
* Click on the `Run Query` button
* Open your console to see the result

## Fixes
#5 
#1 
#20 
#4
